### PR TITLE
ERM-352: Error message if file is too large

### PR DESCRIPTION
**Pull request by CalamityC**
* distinguish upload error from other in FileUploaderField
* add renderUploadError method in FileUploader
* add @folio/react-intl-safe-html to dependencies in package.json
* update CHANGELOG.md

### CODE REVIEW

File: lib/FileUploader/FileUploader.js [hunk 1]
```diff
@@ -58,6 +60,38 @@ export default class FileUploader extends React.Component {
     );
   };
 
+  renderUploadError = () => {
+    const { fileName, isDropZoneActive } = this.props;
+    return (
+      <MessageBanner data-test-error-msg type="warning" hidden={isDropZoneActive} icon={null}>
```
[the remark this answers is not in the document]
Comment (CalamityC) [reply lib/FileUploader/FileUploader.js]: Yes, but the icon on top of the message should be the "document". I could pass that to the MessageBanner too, but it looked odd, especially when i tried to add the second icon (exclamation mark). but i try it again when i made your other suggested changes

[the remark this answers is not in the document]
Comment (CalamityC) [reply lib/FileUploader/FileUploader.js]: I left it as it was, had no success, looked really odd

File: lib/FileUploader/FileUploader.js [hunk 2]
```diff
@@ -58,6 +60,38 @@ export default class FileUploader extends React.Component {
...
+            <Col md={1} />
+            <Col md={11}><SafeHTMLMessage id="stripes-erm-components.errors.uploadError.2" values={{ number: 200 }} /></Col>
+          </Row>
+        </Layout>
+        <Layout className="padding-bottom-gutter">
+          <Row>
+            <Col md={1} />
+            <Col md={11}><SafeHTMLMessage id="stripes-erm-components.errors.uploadError.3" /></Col>
+          </Row>
+        </Layout>
```
[the remark this answers is not in the document]
Comment (aditya-matukumalli) [reply lib/FileUploader/FileUploader.js]: Ah okay, I see it now. Thanks

File: lib/FileUploader/FileUploader.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aditya-matukumalli) [reply lib/FileUploader/FileUploader.js]: Ah okay. I see it now. Thanks

**Pull request merged**